Automattic/wp-calypso
# More Informative Scan: Threat List Page And Fix Dialogs

#### Changes proposed in this Pull Request

This updates the threat list page for the More Informative Scan project(pbuNQi-1IT-p2).

Design reference (p6TEKc-5tS-p2)

I'm not much of a front end developer, so I'm sure that I'm doing things wrong the scss side of things and there are some details I might have missed. Please pay extra attention to those items when reviewing 😅 

#### Testing instructions



* Set up a JP site with Scan and add a few threats (You can use the threat tester plugin to add a few innocuous threats for testing).
* Go to the scan section of both Calypso Blue and Calypso Green (To test Calypso Green you need to run yarn start start-jetpack-cloud and then go to http://jetpack.cloud.localhost:3000 on your browser) and check out the threat listing page.
* Verify that the layout matches the design specs linked above.
* I added the modifications behind a feature flag, as an additional test turn the flag off and verify that the design matches the current production design.
#### Screenshots

##### Before

<img width="772" alt="Screen Shot 2021-11-22 at 23 14 38" src="https://user-images.githubusercontent.com/12788275/142961929-502e9b83-4eda-4f34-919f-081d7ddc9c21.png">

##### After
<img width="747" alt="Screen Shot 2021-11-22 at 23 25 29" src="https://user-images.githubusercontent.com/12788275/142962633-9b78cf57-eba5-4ff8-89ba-9e85c50db3c0.png">
